includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -42,6 +42,8 @@ public function sanitize() {
 				continue;
 			}
 
+			// TODO: add a filter to kill the iframe if not https
```

Comment:
For this, it'd be nice to use a `fallback` element instead that links out to the embed so it's still accessible. I've opened https://github.com/ampproject/amphtml/issues/2433 to see if it's supported for `amp-iframe`.